EdFi.Ods.Utilities.Migration/logistics/scripts/modules/build-edfi-standard-descriptors.psm1

```diff
@@ -0,0 +1,148 @@
+
+    return $script:result | Format-Table
+}
+
+function Remove-TempDirectory([hashtable] $config) {
+    if (-not (Test-Path $config.OutputPath)) { return }
+
+    Remove-Item $config.OutputPath -Force -Recurse
+
+    Write-Host "Removed Temp Directory: " -NoNewLine
+    Write-Host $config.OutputPath -ForegroundColor DarkGray
+}
+
+function New-TempDirectory([hashtable] $config) {
+    $config.OutputPath = New-Item -Force $config.OutputPath -ItemType Directory
+
+    Write-Host "New Temp Directory: " -NoNewLine
+    Write-Host $config.OutputPath -ForegroundColor Green
+
+    return $config.OutputPath
+}
+
+function Copy-PackageFiles([hashtable] $config) {
+    foreach ($file in $config.PackageFiles) {
+        New-Item -Path $file.target -ItemType "Directory" -Force
+        $items = Get-ChildItem $file.src -Recurse
+
+        foreach ($item in $items) {
+            Write-Host "copy $($item.Fullname)" -ForegroundColor DarkGray
+
+            $destination = Copy-Item -Path $item.Fullname -destination $file.target -Force -PassThru
+
+            Write-Host '  -> ' -NoNewLine -ForegroundColor DarkGray
+            Write-Host $destination.FullName -ForegroundColor Green
+        }
+        Write-Host
+        Write-Host "$($items.Length) items copied."
+        Write-Host
+    }
+}
+
+function New-PackageNuspec([hashtable] $config) {
+    $nuspecPath = Join-Path $config.OutputPath 'package.nuspec'
+
+    $params = @{
+        nuspecPath = $nuspecPath
+        id = '$id$'
+        title = '$title$'
+        description = '$description$'
+        version = '$version$'
+        authors = '$authors$'
+        copyright = '$copyright$'
+        owners = '$owners$'
+        requireLicenseAcceptance = $false
+        forceOverwrite = $true
+    }
+    New-Nuspec @params
+
+    function Add-Files([string] $nuspecPath) {
+        [xml]$xml = Get-Content $nuspecPath
+
+        $file = $xml.CreateElement('file')
+
+        $file.SetAttribute('src', '**/*')
+        $file.SetAttribute('target', 'contentFiles/')
+
+        $filesElem = $xml.GetElementsByTagName('files')[0]
+        $filesElem.AppendChild($file)
+
+        $xml.Save($nuspecPath)
+    }
+
+    function Add-ContentFiles([string] $nuspecPath) {
+        [xml]$xml = Get-Content $nuspecPath
+
+        $files = $xml.CreateElement('files')
+
+        $files.SetAttribute('include', 'any/netcoreapp3.1/**/*')
+        $files.SetAttribute('buildAction', 'None')
+        $files.SetAttribute('copyToOutput', 'true')
+
+        $metadata = $xml.GetElementsByTagName('metadata')[0]
+        $contentFiles = $xml.CreateElement('contentFiles')
+        $metadata.AppendChild($contentFiles)
+        $contentFiles.AppendChild($files)
+
+        $xml.Save($nuspecPath)
+    }
+
+    Add-Files $nuspecPath
+    Add-ContentFiles $nuspecPath
+
+    Write-Host "Nuspec Created: " -NoNewline
+    Write-Host (Get-ChildItem $nuspecPath).FullName -ForegroundColor Green
+    Write-Host
+}
```

Comment:
I think `New-Nuspec` is the only functionality coming from package.psm1 here, we can probably just create a .nuspec file and check it in to break this dependency

Reply:
yes , I did it  . Thank you !  EdFi.Standard.Descriptors.nuspec added